clients/client-python/tests/integration/config.py

```diff
@@ -0,0 +1,2 @@
+class Config:
```

Comment:
I think you miss the license header for this file. Please add a license header like other files.

Reply:
Thank you for the heads-up. I have fixed the licensing issue.